app/uk/gov/hmrc/gform/testonly/TestOnlyController.scala

```diff
@@ -852,4 +865,37 @@ class TestOnlyController(
     new GovukTable()(table)
   }
 
+  def snapshotDeleteConfirmation(snapshotId: SnapshotId, backUrl: String) =
+    controllerComponents.actionBuilder.async { implicit request =>
+      import i18nSupport._
+      implicit val lang: LangADT = LangADT.En
+      val actionUrl = uk.gov.hmrc.gform.testonly.routes.TestOnlyController.deleteSnapshot(snapshotId).url
+      Ok(
+        snapshot_delete_confirmation(
+          frontendAppConfig,
+          snapshotId,
+          actionUrl,
+          backUrl
+        )
+      ).pure[Future]
+    }
+
+  def deleteSnapshot(snapshotId: SnapshotId) =
+    controllerComponents.actionBuilder.async { implicit request =>
+      import SnapshotForms._
```

Comment:
Would it be possible to delete all duplications of this line and put a single import
```
import uk.gov.hmrc.gform.testonly.SnapshotForms._
```
at the top instead?